charts/renku/requirements.lock

```diff
@@ -3,14 +3,14 @@ dependencies:
   repository: https://swissdatasciencecenter.github.io/helm-charts/
   version: 0.10.3-3a30de0
 - name: renku-ui
-  repository: https://swissdatasciencecenter.github.io/helm-charts/
-  version: 0.9.2-4538959
+  repository: file:///Users/ableuler/renku/src/renku-ui/helm-chart/renku-ui/
```

Comment:
I guess this was added by mistake :smile: 
I would remove both `requirements` files from this PR since the version numbers will automatically be updated when the repositories' PR are merged

Reply:
Oops, it was indeed. No I wanted to put it there to make testing easier for people who just want to check if deployment works with prebuilt charts. I'll fix it.